features/lwipstack/LWIPStack.h

```diff
@@ -80,6 +80,12 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
          */
         virtual nsapi_connection_status_t get_connection_status() const;
 
+        /** Return netif interface name
+         *
+         * @return       netif name  eg "en0"
+        */
+        virtual char *get_interface_name(char *buf);
```

Comment:
Should this have buffer length as a parameter, also is returning the buffer needed?